nginx/hellogov.conf

```diff
@@ -0,0 +1,19 @@
+server {
+  listen 80;
+  listen [::]:80;
+  server_name hellogov.org;
```

Comment:
Should this now be heyagov.org?

I wonder if (without having chef/ansible/puppet/etc) we need to create a heyagov.conf and a hellogov.conf?